integrations/easyudSDKjs/easyudSDKjs.js

```diff
@@ -496,6 +496,21 @@ function udSDKJS_ServerProjectSync() {
   });
 }
 
+//!
+//! Get a list of collaboration users within the same project
+//!
+//!   json udSDKJS_GetCollabUsers()
+//!
+//! @return {object} An object of all other collab users.
+//! @note {number} You can only get other collaboration users if you have connected with udSDKJS_CreateShared() or udSDKJS_CreateFrom_udCloud().
+//!
+function udSDKJS_GetCollabUsers(additionalInfo) {
+  return new Promise(function (onSuccess) {
+    let onSuccessPtr = Module.addFunction(onSuccess, 'vi');
+    JSON.parse(udSDKJS_GetCollabUsersInternal(additionalInfo ? 1 : 0, onSuccessPtr));
```

Comment:
This `JSON.parse` is doing nothing, except maybe causing an exception to be thrown because `udSDKJS_GetCollabUsersInternal` doesn't directly return anything.

Reply:
fixed